example/mirroring/web/main.dart

```diff
@@ -0,0 +1,25 @@
+library scissors_angular2_example.main;
+
+import 'messages_all.dart';
+import 'package:angular2/bootstrap.dart';
+
+import 'package:scissors_angular2_example/foo.dart';
+import 'package:intl/intl.dart';
+import 'dart:html';
+
+
+main() async {
+  var languageRx = new RegExp(r'\bhl=(\w+)\b');
+  Intl.defaultLocale = languageRx.firstMatch(window.location.href)?.group(1) ??
+      Intl.getCurrentLocale();
+  await initializeMessages(Intl.getCurrentLocale());
+
+  document.querySelector('body').dir =
```

Comment:
Could you leave a TODO(ochafik): add comments here and in permutations (or add some comments yourself?)